Ladybird/Qt/Tab.cpp

```diff
@@ -390,8 +390,9 @@ Tab::Tab(BrowserWindow* window, WebContentOptions const& web_content_options, St
     };
 
     view().on_resize_window = [this](auto const& size) {
-        m_window->resize(size.width(), size.height());
-        return Gfx::IntSize { m_window->width(), m_window->height() };
+        auto new_window_size = m_window->size() - m_view->size() + QSize { size.width(), size.height() };
+        m_window->resize(new_window_size);
+        return Gfx::IntSize { m_view->width(), m_view->height() };
```

Comment:
I've changed this function to resize the viewport to the given size rather than the window, but it seems that the caller of this function expects the window to be set to the given size:

https://github.com/LadybirdBrowser/ladybird/blob/3c897e7cf3594f02f559599e1bf28747c9edba13/Userland/Services/WebContent/WebDriverConnection.cpp#L670-L672

I'm not sure what to do here seeing as there are WPT tests that rely on the new behavior. Running the tests in headless mode would probably be the best idea.